build-logic/buildquality/src/main/kotlin/gradlebuild.ci-reporting.gradle.kts

```
@@ -96,13 +91,13 @@ fun verifyTestFilesCleanup(failedTasks: List<Task>, tmpTestFiles: List<File>) {
     }
 }
 
-fun prepareReportsForCiPublishing(failedTasks: List<Task>, executedTasks: List<Task>, tmpTestFiles: List<File>) {
-    val failedTaskCustomReports = failedTasks.flatMap { it.failedTaskGenericHtmlReports() }
+fun prepareReportsForCiPublishing(testTasksToCollectReports: List<Task>, executedTasks: List<Task>, tmpTestFiles: List<File>) {
```

```suggestion
fun prepareReportsForCiPublishing(tasksToCollectReports: List<Task>, executedTasks: List<Task>, tmpTestFiles: List<File>) {
```

These are not only test tasks.

---

Done.